Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp

```diff
@@ -195,6 +196,8 @@ PropertyID property_id_from_string(StringView string)
     });
 
     generator.append(R"~~~(
+    if (string.starts_with("--"sv) && string != "--")
+        return PropertyID::Custom;
```

Comment:
If we are not going to filter for now should we at least add a FIXME or something?
With a spec link perhaps.

Reply:
There is no filtering to be done because the parser appends the code point of any escape sequence to its result while consuming an identifier ([spec](https://drafts.csswg.org/css-syntax-3/#consume-name)). This means that, for example, the identifier `--tes\ t` will be stored as `--tes t` internally. Whenever dealing with such identifiers, we should be using the internal representation.